apax/cli/templates/md_config_minimal.yaml

```diff
@@ -1,7 +1,9 @@
 ensemble:
   name: nvt
   dt: 0.5 # fs time step
-  temperature: <T> # K
+  temperature_schedule:
+    name: constant
+    T0: <T> # K
```

Comment:
I think we should document all the possible config inputs. Not only for the temp scheduler.

Reply:
you mean with regards to the unit?

Reply:
No I mean in various parts of the config one can choose different temp schedules, ensemble types and so on. We only communicate one of these choices properly. I think we should think about a better way of communicating the different choices that can be made in the configuration script.

Reply:
This does not be addressed here in the PR I just recognised it.